exportoptionsgenerator/archive_info_provider.go

```diff
@@ -0,0 +1,58 @@
+package exportoptionsgenerator
```

Comment:
Do we really need one more xcarchive reader? We have one reader on V1 branch, and two others on V2 already.

Reply:
Converted this to a simple function that uses v1 archive.
Removed one of the v2 xcarchive readers.